nikola/feedutil.py

```diff
@@ -0,0 +1,463 @@
+# -*- coding: utf-8 -*-
+
+# Copyright © 2015 IGARASHI Masanao
+
+# Permission is hereby granted, free of charge, to any
+# person obtaining a copy of this software and associated
+# documentation files (the "Software"), to deal in the
+# Software without restriction, including without limitation
+# the rights to use, copy, modify, merge, publish,
+# distribute, sublicense, and/or sell copies of the
+# Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice
+# shall be included in all copies or substantial portions of
+# the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY
+# KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE
+# WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR
+# PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS
+# OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR
+# OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+# SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+"""Utility functions for feed"""
+
+from __future__ import unicode_literals
+import os
+import sys
+try:
+    from urlparse import urljoin
+except ImportError:
+    from urllib.parse import urljoin  # NOQA
+import io
+import uuid
+import mimetypes
+from datetime import datetime
+import dateutil.tz
+import lxml.html
+from lxml.html import fragment_fromstring
+import lxml.etree
+import html5lib
+from feedgen.feed import FeedGenerator
+from .image_processing import ImageProcessor
+
+from . import utils
+
+
+xml_dec_line = '<?xml version="1.0" encoding="utf-8"?>\n'
+xsl_line = '<?xml-stylesheet type="text/xsl" href="{0}" media="all"?>\n'
+
+class FeedUtil(object):
+
+    """The utility class for feed."""
+
+    def __init__(self, site):
+        """Setup."""
+        self.site = site
+
+    def atom_renderer(self, fg, output_path, atom_path, xsl_stylesheet_href,
+                      pretty=True):
```

Comment:
This does not need to be an option here nor in rss_renderer(). Always make this pretty, and let users customize the output with filters (like `xmlminify`).